src/Jobs/GenerateImageAnnotationPatch.php

```diff
@@ -27,13 +24,7 @@ public function handleFile(VolumeFile $file, $path)
 
         $buffer = $this->getAnnotationPatch($image, $this->annotation->getPoints(), $this->annotation->getShape());
 
-        $svgTargetPath = str_replace(config('largo.patch_format'),'svg',$targetPath);
-        $svgAnnotation = $this->getSVGAnnotationPatch($image->width,$image->height,
-                                        $this->annotation->getPoints(), $this->annotation->getShape());
-        
-        
         Storage::disk($this->targetDisk)->put($targetPath, $buffer);
-        Storage::disk($this->targetDisk)->put($svgTargetPath, $svgAnnotation);
```

Comment:
This would now no longer generate an SVG when an annotation was created, right? But we still need that.